File: OSSA/Lab Sheets/Lab1/exe10.c
//ps -al -> to view the all processes process table
//Zombi Process
#include <stdio.h>
int main()
{
        int id;
        if ((id = fork()) == 0)
        {
                printf("I am child process \n");
                printf("pid : %d\n", getpid()); //child process have to exit so it has to go to parent process but parent process is in sleep
                //then execution suspened(parent) because child cannot exit
        }
        else
        {
                while (1)
                        sleep(100);
        }
}

File: OSSA/Lab Sheets/Lab1/exe10.go
package main

import (
	"fmt"
	"os"
	"time"
)

func main() {
	go func() {
		fmt.Printf("I am child process \n")
		fmt.Printf("pid : %d\n", os.Getpid())
	}()
	for {
		time.Sleep(time.Microsecond * 20)
	}
}
